pkg/cmd/workspace/code.go

```diff
@@ -107,11 +108,45 @@ var CodeCmd = &cobra.Command{
 		}
 
 		providerMetadata := ""
+
+		var yesFlag bool
+		form := huh.NewForm(
```

Comment:
Since this form is reused here and in the ssh command, it would be nice to see it in the `views` pkg and simply imported.

It will make it easier to maintain.

Reply:
moved the form to views as suggested